test/intl402/DateTimeFormat/prototype/formatRangeToParts/fractionalSecondDigits.js

```diff
@@ -27,36 +28,32 @@ const d1 = new Date(2019, 7, 10,  1, 2, 3, 234);
 const d2 = new Date(2019, 7, 10,  1, 2, 3, 567);
 const d3 = new Date(2019, 7, 10,  1, 2, 13, 987);
 
-assert.throws(RangeError, () => {
-    new Intl.DateTimeFormat(
-      'en', { minute: "numeric", second: "numeric", fractionalSecondDigits: 0});
-  }, "fractionalSecondDigits 0 should throw RangeError for out of range");
```

Comment:
I'd recommend keeping these out-of-range tests but changing the values from 0 and 4, to -1 and 10.